modules/candidate_list/templates/menu_candidate_list.tpl

```diff
@@ -250,7 +251,7 @@
                 {/if}
                 {if $items[item][piece].DCCID != "" AND $items[item][piece].name == "PSCID"}
                     {assign var="PSCID" value=$items[item][piece].value}
-                    <a href="{$baseurl}/main.php?test_name=timepoint_list&candID={$items[item][piece].DCCID}">{$items[item][piece].value}</a>
+                    <a href="{$baseurl}/main.php?test_name=timepoint_list&candID={$items[item][piece].DCCID}" id="pscid">{$items[item][piece].value}</a>
```

Comment:
The class variable should be used instead of the id variable. This works in the way you access the element, but it is better practice to keep the id variable unique to the document element.